spec/data_helper.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+def api_get_call_contacts
```

Comment:
Can we have that file in a less ugly version? Something like a pretty hash with a to_json for example, or anythink readable